src/js/game/LevelMVC/LevelView.js

```diff
@@ -2001,6 +2004,22 @@ export default class LevelView {
         });
         break;
 
+      //pelican
+      case "redstone_dust":
+        atlas = this.blocks[blockType][0];
+        frame = this.blocks[blockType][1];
+        xOffset = this.blocks[blockType][2];
+        yOffset = this.blocks[blockType][3];
+        sprite = plane.create(xOffset + 40 * x, yOffset + plane.yOffset + 40 * y, atlas, frame);
+        frameList = Phaser.Animation.generateFrameNames("TNTexplosion", 0, 8, "", 0);
+        sprite.animations.add("explode", frameList, 7, false).onComplete.add(() => {
+          this.playExplosionCloudAnimation([x, y]);
+          sprite.kill();
+          this.toDestroy.push(sprite);
+          this.actionPlaneBlocks[this.coordinatesToIndex([x, y])] = null;
+        });
+        break;
+
```

Comment:
I don't think you need this section at all.  The `default:` case on line 2023 should account for the added block type.